langchain/vectorstores/redis.py

```diff
@@ -33,36 +33,10 @@
     from redis.commands.search.query import Query
 
 
-# required modules
-REDIS_REQUIRED_MODULES = [
-    {"name": "search", "ver": 20400},
-    {"name": "searchlight", "ver": 20400},
-]
-
 # distance mmetrics
 REDIS_DISTANCE_METRICS = Literal["COSINE", "IP", "L2"]
 
 
-def _check_redis_module_exist(client: RedisType, required_modules: List[dict]) -> None:
```

Comment:
This is not something we should remove. This checks for the existence of the required Redis extensions on the database side (RediSearch: https://redis.io/docs/stack/search).